sdk/cosmos/azure_data_cosmos/src/clients/container_client.rs

```diff
@@ -35,7 +35,7 @@ impl ContainerClient {
         database_link: &ResourceLink,
         container_id: &str,
     ) -> Self {
-        let link = database_link
+        let link: ResourceLink = database_link
```

Comment:
Why?

Reply:
Yeah, this type annotation shouldn't be needed! The type of `link` should be apparent to the compiler from the return type of `.item`. You can just revert this back to how it was before :)